samples/core/train_until_good/train_until_good.py

```diff
@@ -0,0 +1,88 @@
+# This sample demonstrates continuous training using a train-eval-check recursive loop.
```

Comment:
Shall we add copyright statement at the top?

Reply:
Added the license header.